README.md

```diff
@@ -1,6 +1,10 @@
-test-editor-xtext
+DEPRECATED  test-editor-xtext
 =================
 
+:warning: **WARNING** This project is deprecated because the Test-Editor Team decided to cut off the RCP functionality  for the sake of the Test-Editor-Web in October 2018 :warning:
+
+The new backend repository for the project :star:Test-Editor-Web:star: can be found under [test-editor-backend](https://github.com/test-editor/test-editor-backend) 
```

Comment:
While that's true, this project has a direct replacement in [test-editor-xtext-gradle](https://github.com/test-editor/test-editor-xtext-gradle), so shouldn't we refer to that project, instead?